update.sh

```diff
@@ -14,5 +19,23 @@ fi
 git submodule sync
 git submodule update --init --recursive
 
-git submodule foreach python setup.py develop --user
-git submodule foreach pip install --user --editable .
+for submodule in core web-api france parsers
+do
+	cd openfisca/$submodule/
+	python setup.py develop $user_option
+	pip install $user_option  --editable .
+	cd ../..
+done
+
+for old_symlink_extension in $(find openfisca/france/openfisca_france/model/extensions -type l)
+do
+	rm $old_symlink_extension
+done
+
+for extension in $(ls  -d openfisca/extensions/*)
+do
+	if [[  $(git submodule | grep $extension) ]]
```

Comment:
Why `test` the result of a subshell execution rather than simply checking the exit value?